ssh/ssh.go

```diff
@@ -30,20 +30,20 @@ type SSHMetadata struct {
 	Path        string
 }
 
-func FormatArgs(cmd string, args []string, needShell bool) (string, []string) {
+func FormatArgs(cmd string, args []string, needShell bool, multiplex bool) (string, []string) {
```

Comment:
I see the `multiplex` argument added here but I don't see it being used ... am I missing something here?

Reply:
No, other than in the tests and some of the code we pass all of the arguments directly from the return value of another function into this function, and it makes it a lot easier to not update all the tests to just pass the extra argument and not strip it off.

Reply:
Got it, thanks!